infra/build_specified_commit.py

```diff
@@ -77,20 +79,25 @@ def detect_main_repo(project_name, repo_name=None, commit=None, src_dir='/src'):
     src_dir: The location of the projects source on the docker image.
 
   Returns:
-    The repo's origin, the repo's name.
+    The repo's origin, the repo's path.
   """
-  # TODO: Add infra for non hardcoded '/src'.
+
   if not repo_name and not commit:
     print('Error: can not detect main repo without a repo_name or a commit.')
     return None, None
   if repo_name and commit:
     print('Both repo name and commit specific. Using repo name for detection.')
 
-  helper.build_image_impl(project_name)
+  # Change to oss-fuzz main directory so helper.py runs correctly.
+  utils.chdir_to_base()
+  if not helper.build_image_impl(project_name):
+    print('Error: building {} image failed.'.format(project_name),
```

Comment:
We should get rid of all prints and change it to log calls in a future CL.

Reply:
Noted.